docker-compose.dev.yaml

```diff
@@ -20,7 +20,7 @@ services:
       context: .
       dockerfile: Dockerfile.dev
     ports:
-      - 4000:4000
+      - "${SERVER_PORT:-4000}:${SERVER_PORT:-4000}"
```

Comment:
Is `${SERVER_PORT}` the port that the external Admin instance will communicate with?
I'm assuming that Docker will automatically insert this value based on the environment variable value. Correct?

Reply:
Yes!